### PR TITLE
Include the CA profile when modifying request in ipa_certupdate

### PR DESCRIPTION
**Pull request by rcritten**
The CA tracking request is modified (it calls renew but it doesn't
actually do a renewal) as part of ipa-certupdate and it dropped
the profile. ipa-healthcheck discovered this condition.

https://pagure.io/freeipa/issue/8644

Signed-off-by: Rob Crittenden <rcritten@redhat.com>

### CODE REVIEW

File: ipaclient/install/ipa_certupdate.py [hunk 1]
```diff
@@ -188,7 +188,8 @@ def update_server(certs):
         #
         logger.debug("resubmitting certmonger request '%s'", request_id)
         certmonger.resubmit_request(
-            request_id, ca='dogtag-ipa-ca-renew-agent-reuse', profile='')
+            request_id, ca='dogtag-ipa-ca-renew-agent-reuse',
+            profile='caCACert')
```
Comment (frasertweedale) [inline ipaclient/install/ipa_certupdate.py:192]: [nit] Will it still work properly if we omit the `profile` kwarg?

**Review by frasertweedale — COMMENTED** [review]
Nice that healthcheck is finding issues for us now :)

One minor question (see comments).

Otherwise, ACK after temp commit green and temp commit removed.

File: ipaclient/install/ipa_certupdate.py (hunk 1 above)
Comment (rcritten) [reply ipaclient/install/ipa_certupdate.py:192]: It looks like it will. Let me drop that. There is no need to override things here. Nice catch.

**Comment by rcritten** [comment]
The test failed on uninstall with the known dbus issue.

**Comment by flo-renaud** [comment]
@rcritten the test is probably failing because it's executed after `test_ipa_healthcheck_expiring` which is changing the date back and forth. I would move the new test before `test_ipa_healthcheck_expiring`.

**Comment by rcritten** [comment]
I need to move this up further in the list. It failed because of a revoked cert in another test.

**Comment by flo-renaud** [comment]
Adding re-run, the test failed in uninstall (known issue https://pagure.io/freeipa/issue/8506)

**Comment by rcritten** [comment]
Re-run, test timeout. It's possible that with the new test + ipa-certupdate I'll need to extend the timeout.

**Comment by flo-renaud** [comment]
LGTM, you can remove the temp commit

**Comment by rcritten** [comment]
Temp commit removed.

**Comment by rcritten** [comment]
The build failed, http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/16176060-505c-11eb-80e2-fa163e068b94

2021-01-06 20:19:47,169    DEBUG  Package libuv-1.40.0-1.fc32.x86_64.rpm is not signed
2021-01-06 20:19:47,169    DEBUG  Error: GPG check FAILED

**Comment by flo-renaud** [comment]
master:

* 53f4fd91269dac89d83d255b174e6c0344c644f1 Don't change the CA profile when modifying request in ipa_certupdate
* 8e9fecd72ac1db3969583207e549d2bdadc20986 ipatests: test that no errors are reported after ipa-certupdate

**Pull request closed without merging**